test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -15,25 +15,158 @@ namespace winrt
     using namespace winrt::Windows::Foundation;
 }
 
-bool BackgroundActivationTest() // Activating application for background test.
+winrt::ToastNotification CreateToastNotification(winrt::hstring message)
 {
-    return true;
+    winrt::hstring xmlPayload{ L"<toast>" + message + L"</toast>" };
+
+    winrt::XmlDocument xmlDocument{};
+    xmlDocument.LoadXml(xmlPayload);
+
+    return winrt::ToastNotification(xmlDocument);
 }
 
-bool UnregisterBackgroundActivationTest()
+winrt::ToastNotification CreateToastNotification()
 {
-    winrt::ToastNotificationManager::Default().UnregisterActivator();
+    return CreateToastNotification(L"intrepidToast");
+}
+
+void EnsureNoActiveToasts()
+{
+    auto removeAllAsync = winrt::ToastNotificationManager::Default().RemoveAllAsync();
```

Comment:
Please make sure that the WinRT API call to REmoveAll succeeds even if there is nothing to remove. It should not throw.